configure.ac

```
@@ -6097,6 +6094,12 @@ PY_STDLIB_MOD_SIMPLE([math], [], [$LIBM])
 dnl needs libm and on some platforms librt
 PY_STDLIB_MOD_SIMPLE([_datetime], [], [$TIMEMODULE_LIB $LIBM])
 
+dnl needs libbsd on some platforms
+PY_STDLIB_MOD([fcntl], [],
+              m4_flatten([test "$ac_cv_header_sys_ioctl_h" = "yes"
+                            -a "$ac_cv_header_fcntl_h"     = "yes"]),
+              [], [$FCNTL_LIBS])
```

Let's keep the indention and formatting consistent with other blocks:

```suggestion
PY_STDLIB_MOD([fcntl],
  [], [test "$ac_cv_header_sys_ioctl_h" = "yes" -a "$ac_cv_header_fcntl_h" = "yes"],
  [], [$FCNTL_LIBS])
```

---

👍🏻